include/seqan3/io/record.hpp

```diff
@@ -108,7 +110,7 @@ enum class field
     SEQ SEQAN3_DEPRECATED_310 = seq, //!< Please use the field name in lower case.
     ID SEQAN3_DEPRECATED_310 = id, //!< Please use the field name in lower case.
     QUAL SEQAN3_DEPRECATED_310 = qual, //!< Please use the field name in lower case.
-    SEQ_QUAL SEQAN3_DEPRECATED_310 = seq_qual, //!< Please use the field name in lower case.
+    SEQ_QUAL SEQAN3_DEPRECATED_310 = seq_qual, //!< [DEPRECATED] Please use the field name in lower case.
```

Comment:
and

```suggestion
    SEQ_QUAL SEQAN3_DEPRECATED_310 = _seq_qual_deprecated, //!< [DEPRECATED] Please use the field name in lower case.
```

will enable us to remove the surrondung `#pragma GCC diagnostic ignored "-Wdeprecated-declarations"`

Reply:
I believe this is a matter of style and taste. To me both solution are valid solution.
I think that the `pragma` solution is very spot on, but the `_seq_qal_deprecated` is very compact.
You decide :-)

Reply:
True, I've made the experience that opening  `#pragma GCC diagnostic ...` can hide errors if you span it around complete classes / structs. So I would prefer a more local solution. Can you change it? :)

Reply:
Done :-)